plasma_framework/docs/design/ald-exit-game-definitions-and-properties.md

```diff
@@ -0,0 +1,133 @@
+# ALD Exit Game Definitions And Properties
+
+## New definitions from ALD (abstract layer design)
+Based on [MoreVP document](https://github.com/omisego/elixir-omg/blob/master/docs/morevp.md). This section adds some new definitions introduced from ALD or give explicit names to some of the implicit characteristics of the origin MoreVP spec.
+
+### SpendingCondition Fulfilled
+1. Checks an input tx and the spending tx, whether the spending tx fulfil the requirement of spending the output of input tx or not.
+1. Basic checks:
+    - Spending tx input points to the output of input tx.
+    - Spending tx fulfil the “lock” condition of the output
```

Comment:
What is the lock condition

Reply:
Changed the description a bit. But "lock" is from bitcoin, the lock and unlock script idea.